steve/util.py

```diff
@@ -378,10 +378,14 @@ def out(*output, **kw):
     sys.stdout.write(output + '\n')
 
 
-def load_json_files(config):
+def load_json_files(config, path=None):
```

Comment:
I think this can go at some point, but I did not want to break usage of load_json_files in other places, nor
wanted to have to check all usage points right now. I would tend to have the function return a triple, with full_path_name, base_name, and data, and have the caller use what is necessary. That is independent from where the json files are found. A rootdir parameter doesn't help with making `steve-cmd status` not showing the full path to the json file.